code/modules/projectiles/guns/energy.dm

```diff
@@ -103,13 +103,15 @@
 		exam_proj = for_ammo.projectile_type
 		if(!ispath(exam_proj))
 			continue
-
-		if(initial(exam_proj.damage) > 0) // Don't divide by 0!!!!!
-			readout += "[span_warning("[HITS_TO_CRIT(initial(exam_proj.damage) * for_ammo.pellets)] shot\s")] on [span_warning("[for_ammo.select_name]")] mode before collapsing from [initial(exam_proj.damage_type) == STAMINA ? "immense pain" : "their wounds"]."
-			if(initial(exam_proj.stamina) > 0) // In case a projectile does damage AND stamina damage (Energy Crossbow)
-				readout += "[span_warning("[HITS_TO_CRIT(initial(exam_proj.stamina) * for_ammo.pellets)] shot\s")] on [span_warning("[for_ammo.select_name]")] mode before collapsing from immense pain."
+		if(projectile_damage_multiplier)
+			if(initial(exam_proj.damage) > 0) // Don't divide by 0!!!!!
+				readout += "[span_warning("[HITS_TO_CRIT((initial(exam_proj.damage) * projectile_damage_multiplier) * for_ammo.pellets)] shot\s")] on [span_warning("[for_ammo.select_name]")] mode before collapsing from [initial(exam_proj.damage_type) == STAMINA ? "immense pain" : "their wounds"]."
+				if(initial(exam_proj.stamina) > 0) // In case a projectile does damage AND stamina damage (Energy Crossbow)
+					readout += "[span_warning("[HITS_TO_CRIT((initial(exam_proj.stamina) * projectile_damage_multiplier) * for_ammo.pellets)] shot\s")] on [span_warning("[for_ammo.select_name]")] mode before collapsing from immense pain."
+			else
+				readout += "a theoretically infinite number of shots on [span_warning("[for_ammo.select_name]")] mode."
 		else
-			readout += "a theoretically infinite number of shots on [span_warning("[for_ammo.select_name]")] mode."
+			readout += "a theoretically infinite number of shots on [span_warning("[for_ammo.select_name]")] mode due to esoteric or nonexistent offensive potential."
```

Comment:
Ditto if the projectile's damage is less than zero or if the multiplier is 0 or less.

Reply:
couldn't really do this because i didn't want to change how the damage and stamina check was structured for energy weapons too much - if it's that bad i guess i can take a second look.